mobile/src/main/java/ch/epfl/sdp/mobile/ui/play/PlayScreen.kt

```diff
@@ -26,24 +25,61 @@ fun PlayScreen(
     modifier: Modifier = Modifier,
     contentPadding: PaddingValues = PaddingValues(),
 ) {
+  val buttonState = rememberExpandableFloatingActionButtonState()
+
   Box(modifier = modifier.fillMaxSize().padding(contentPadding)) {
-    NewGameButton(onNewGame = state.onNewGameClick, modifier = Modifier.align(Alignment.BottomEnd))
+    NewGameButton(
+        state = buttonState,
+        onLocalGame = { state.onLocalGameClick() },
+        onRemoteGame = { state.onOnlineGameClick() },
+        modifier = Modifier,
+    )
   }
 }
 
 /**
  * Composable that composes a new game button
- * @param onNewGame callable upon button action
+ * @param onLocalGame callable upon button action
  * @param modifier the [Modifier] for this composable.
  */
 @Composable
-fun NewGameButton(onNewGame: () -> Unit, modifier: Modifier = Modifier) {
+fun NewGameButton(
+    state: ExpandableFloatingActionButtonState,
+    onLocalGame: () -> Unit,
+    onRemoteGame: () -> Unit,
+    modifier: Modifier = Modifier,
+) {
   val strings = LocalLocalizedStrings.current
-  ExtendedFloatingActionButton(
-      text = { Text(strings.newGame) },
-      onClick = onNewGame,
-      shape = CircleShape,
-      modifier = modifier.padding(16.dp).height(56.dp),
-      backgroundColor = PawniesColors.Green800,
-      icon = { Icon(PawniesIcons.Add, null) })
+
+  Box(
+      modifier.fillMaxSize().background(MaterialTheme.colors.background),
+      Alignment.BottomEnd,
+  ) {
```

Comment:
The alignment be set in the parent.

Reply:
Fixed in [0a0980e](https://github.com/epfl-SDP/android/pull/226/commits/0a0980ec39e8c6024b8d84dbfb41a6ce3a19de16)